tests/regression_opcodes.rs

```diff
@@ -0,0 +1,276 @@
+//! serde regression tests for blockdata::opcodes module.
```

Comment:
Also better rename the file and use inner cfg.

Reply:
Done, thanks.